src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/customer.html.twig

```diff
@@ -33,7 +33,7 @@
       <div class="col-4">
         <input type="text"
                class="form-control"
-               id="customerSearchInput"
+               id="customer-search-input"
                data-url="{{ path('admin_customers_search') }}"
```

Comment:
Do we still need to define urls in twig?

Reply:
out of this pr scope i thought